backend/app/controllers/controller.js

```diff
@@ -0,0 +1,55 @@
+/* This is the controller file. Its functions are called when 
+a route is navigated to. These functions in turn call upon the 
+database accessor to get the requested information out of the 
+database to return to the front end. */
+
+import Accessor from "../db_accessors/accessor.js";
+
+export default class Controller {
+    /* Function to get every single blog out of the database */
+    static async getBlogs(req, res) {
+        try {
+            const blogs = await Accessor.getAllBlogs();
+            res.json(blogs);
+            return blogs;
+        } catch (e) {
+            console.log("Failed due to:", e);
```

Comment:
For the sake of the server not crashing, every branch of code should result in some sort of HTTP response -- plus, when thinking about frontend, you can set up listeners for a 200 or !200 response, if you set status code to error out here as a res.json

Reply:
When an HTTP request is not met with a direct response, the server hangs and times out / crashes